framework/doc/content/finite_volumes/fv_design.md

```diff
@@ -172,15 +172,31 @@ demonstrated second order convergence with respect to mesh refinement. On a
 non-orthogonal mesh test (see the
 [compact](non-orthogonal/advection-diffusion-reaction.i) and
 [extended](/extended-adr.i) input files) the compact stencil displays second
-order convergence while the extended stencil drops a half order to 1.5 order
-convergence. On a
-[skewed mesh test](fv_adapt/steady-adapt.i) where skewness is introduced via
+order convergence while the vertex-based extended stencil drops a half order to 1.5 order
+convergence. On a [skewed mesh test](fv_adapt/steady-adapt.i) where skewness is introduced via
```

Comment:
this mesh you linked to isnt skewed?

Reply:
The [skewed mesh test] was already there. I didn't modify it. This is the only link I added: 
[skewed](skewness-correction/adv-diff-react/skewed.i) and it is skewed for sure.